spec/features/user_views_past_orders_spec.rb

```diff
@@ -0,0 +1,30 @@
+require "rails_helper"
+
+RSpec.describe "User views past orders" do
+  context "with valid authorization" do
+    let(:user) { User.create(full_name: "example",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.